packages/adminrouter/extra/src/test-harness/modules/generic_test_code/generalised_tests.py

```diff
@@ -189,8 +192,13 @@ def _testdata_to_is_upstream_correct_testdata(tests_config, node_type):
 
         h = x['tests']['is_upstream_correct']
 
+        if 'vhost' in h:
+            vhost = h['vhost']
+        else:
+            vhost = None
```

Comment:
Or `vhost = h.get('vhost')`

Reply:
This comment applies to a few other places.

Reply:
Nice! Thanks!